pkgs/servers/mastodon/version.nix

```diff
@@ -0,0 +1 @@
+"v2.9.3"
```

Comment:
```suggestion
"2.9.3"
```
For consistency with the rest of Nixpkgs.

Reply:
There are some packages with and some without v in nixpkgs, depending on the upstream tags. In the case of mastodon the upstream tags include a v. If you still think it should be changed, please also change the update script to remove the v when updating the release.

Reply:
Agree this shouldn’t have the v.

Reply:
done